README.md

```diff
@@ -1,34 +1,34 @@
-# Cognizant COVID X-Prize
+# Pandemic Response Challenge
```

Comment:
Possibly add `XPRIZE` here and below: `Pandemic Response Challenge` -> 'XPRIZE Pandemic Response Challenge`

Reply:
Done